ci/README.md

```diff
@@ -0,0 +1,48 @@
+# Tekton CI Pipeline
+
+This directory contains the configuration for a tekton pipeline that tests
+the installed `route-monitor-operator` version in the logged in cluster.
```

Comment:
are there other things that need to be installed on the cluster before we can run it? I.e. is Tekton part of it?

Reply:
Is there is a yaml for the subscription, applying this should probably be a first, independent step.

Reply:
subscription to tekton is also defined in [this yaml](https://github.com/openshift/route-monitor-operator/pull/102/files#diff-ffa7e8f720f478144b6ed0d308aa83597c25514617a0fa394a8e3d79e5e497e7). And I would say the same argument as above applies

Reply:
yeah I think I'd prefer just mention applying that subscription first and then the pipeline stuff rather than apply until good ;)